package.json

```diff
@@ -10,7 +10,6 @@
     "start:sdk:local:forcecleanup": "node dev-mode-link-local-sdk.mjs localsdk forcecleanup start",
     "start:sdk:local:nocleanup": "node dev-mode-link-local-sdk.mjs localsdk nocleanup start",
     "reset:sdk:prod": "node dev-mode-link-local-sdk.mjs prodsdk forcecleanup nostart",
-    "build:pr:preview": "node build.mjs preview && NODE_OPTIONS=--max-old-space-size=8192 vite build",
```

Comment:
Your PR not only updates the uiweb package but also:
- Reinstalls all the unused libraries which I have already removed.
- Removes the build:pr:preview command due to which you deploy preview is failing.
- Downgrades the packageManager from 4.2.2 to 3.4.1 in package.json.

Please fix these changes your deploy preview should work after that. I would suggest to revert these changes, delete your node modules and remove the code in yarn.lock and then do a fresh yarn install.